### PR TITLE
Add defender v2.30.0 changelog

### PR DESCRIPTION
**Pull request by collins-w**
- Adds defender v2.30.0 changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/changelog.adoc [hunk 1]
```diff
@@ -1,5 +1,13 @@
 = Changelog
 
+[[release-2024-09-16]]
+== September 16th, 2024
+
+* *General*
+** Fix: Resolved an issue where UI components did not refresh after subscribing to a paid subscription, preventing resource creation when exceeding quota until the page is manually refreshed.
+** Feature: Allow users to submit feedback on code app report issue.
+** Feature: API keys UI improvements.
+
```
Comment (d-carmo) [inline docs/modules/ROOT/pages/changelog.adoc:10]: IMO, missing:
* Fix: Fixed CORS errors when inspecting private network deployments
* Fix: Solved issue where pausing and unpausing notification channels didn't always work

* Feature: Better notification when quotas reached

File: docs/modules/ROOT/pages/changelog.adoc [hunk 2]
```diff
@@ -1,5 +1,16 @@
 = Changelog
 
+[[release-2024-09-16]]
+== September 16th, 2024
+
+* *General*
+** Fix: Resolved an issue where UI components did not refresh after subscribing to a paid subscription, preventing resource creation when exceeding quota until the page is manually refreshed.
```
Comment (tirumerla) [inline docs/modules/ROOT/pages/changelog.adoc:7]: Not applicable for public changelog

File: docs/modules/ROOT/pages/changelog.adoc [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+** Feature: Allow users to submit feedback on code app report issue.
+** Feature: API keys UI improvements.
+** Fix: Fixed CORS errors when inspecting private network deployments.
+** Fix: Solved issue where pausing and unpausing notification channels didn't always work.
```
Comment (tirumerla) [inline docs/modules/ROOT/pages/changelog.adoc:11]:
```suggestion
** Fix: Bug in notification channels
```

**Review by tirumerla — CHANGES_REQUESTED** [review]
lgtm, added few comments

**Review by tirumerla — APPROVED** [review]
lgtm

**Pull request merged**